core/dive.h

```diff
@@ -746,12 +747,12 @@ extern void average_max_depth(struct diveplan *dive, int *avg_depth, int *max_de
  * Note: we have to use the typedef "dive_table_t" instead of "struct dive_table",
  * because MOC removes the "struct", but dive_table is already the name of a global
  * variable, leading to compilation errors. Likewise for "struct trip_table" and
- * "struct dive_site_table". */
+ * "struct dive_site_table" (defined in "divesite.h"). */
+#include <QObject>
```

Comment:
why do we suddenly need to include  `QObject` here?

Reply:
`#include <QObject>` is needed for the metatype-macros.
Previously, `dive.h` included `divesite.h` which includes `QObject`. Now that this is no longer the case, `dive.h` has to include `QObject` explicitly.